uaclient/messages/__init__.py

```diff
@@ -2164,6 +2164,24 @@ def __repr__(self):
     ).format(url=urls.PRO_DASHBOARD),
 )
 
+E_TOKEN_FORBIDDEN_EXPIRED = FormattedNamedMessage(
+    "token-forbidden-expired",
+    t.gettext(
+        """\
+Contract "{{contract_id}}" expired on {{contract_expiry_date}}
+Visit {url} to manage contract tokens."""
+    ).format(url=urls.PRO_DASHBOARD),
+)
```

Comment:
sorry I see you have new messages now

for consistency (and to ease the translation a bit), could we make the existing ones extend those?
something like
```
E_ATTACH_FORBIDDEN_EXPIRED = FormattedNamedMessage(
    "attach-forbidden-expired",
    t.gettext( "Attach denied:")
    + "\n"
    + E_TOKEN_FORBIDDEN_EXPIRED
)
```

Reply:
That's a good suggestion. Will do.

Reply:
Updated, cleaned up commit history as well (also rebased)